.travis.yml

```diff
@@ -4,6 +4,10 @@ language:
 node_js: 
   "10.15.3"
 
+dist: xenial
+services:
+  - xvfb
```

Comment:
why is this guy here?

Reply:
It is a service called virtual framebuffer that help us to learn end to end tests on Travis.